src/index.tsx

```diff
@@ -16,6 +17,10 @@ if (process.env.NODE_ENV === 'production') {
   log.setLevel(log.levels.DEBUG, false);
 }
 
+if (process.env.REACT_APP_SENTRY_DSN) {
+  Raven.config(process.env.REACT_APP_SENTRY_DSN).install();
```

Comment:
This needs to configure `{logger: 'client-js'}` so we can tell it apart from server logs

https://github.com/mozilla/addons-frontend/blob/533c052acc959664baa817c4b69aa76d83b5dc34/src/core/client/base.js#L38

Reply:
Which server logs? Because, we don't log anything on the server.

Reply:
Yes but we may log things on the server in the future now that we have to run the token injection.

Reply:
Ah, ok. My take would be to leave things as is for now and to configure a `{ logger: 'server' }` when we'll need one.